CHANGELOG.md

```diff
@@ -41,6 +42,7 @@ This file is used to list changes made in each version of the AWS ParallelCluste
 - Upgrade NVIDIA Fabric manager to `nvidia-fabricmanager-460`.
 - Install ParallelCluster AWSBatch CLI in dedicated python3 virtual env.
 - Upgrade Python version used in ParallelCluster virtualenvs from version 3.6.13 to version 3.7.10.
+- Reduce default Slurm ResumeTimeout to from 3600 seconds to 1800 seconds
```

Comment:
Minor: can we move this under `Upgrade Slurm to version 20.11.7.` so we can keep all slurm changes together?

Reply:
Moved